webapp/src/app/types/board.model.ts

```diff
@@ -43,6 +43,14 @@ export class Board {
     });
   }
 
+  isCoordOfLastMove(x: number, y: number): boolean {
+    if (this.lastMove === null) return false;
+    return (
+      (this.lastMove.fromCoord.x === x && this.lastMove.fromCoord.y === y) ||
+      (this.lastMove.toCoord.x === x && this.lastMove.toCoord.y === y)
```

Comment:
I think - if lastMove is set correctly in the backend - that the toCoord will always be occupied by the stone that just moved, so it can be removed from here.

Reply:
That hole will be occupied by the stone but also be highlighted to indicate the stone was moved there on the last move.

Reply:
Aaah, now I see - awesome!